Source/KakapoDB.swift

```diff
@@ -92,24 +92,32 @@ public class KakapoDB {
         return object
     }
     
-    public func update<T: Storable>(entity: T) -> Bool {
-        return barrierSync {
+    public func update<T: Storable>(entity: T) throws {
+        let updated: Bool = barrierSync {
```

Comment:
I already tried to rethrow but I was always getting issues with the inner dispatch_barrier_sync to make it work